web/e2e/utils/config.ts

```diff
@@ -10,6 +10,7 @@ export const config = {
   signUpSecret: process.env["REEARTH_WEB_E2E_SIGNUP_SECRET"],
   authProvider: process.env["REEARTH_WEB_AUTH_PROVIDER"],
   developerMode: process.env["REEARTH_WEB_DEVELOPER_MODE"],
+  superAdmins: process.env["REEARTH_WEB_SUPER_ADMINS"],
```

Comment:
I like @keiya01's naming idea for `earlyAccessAdmins` instead. More to the point and keeps the idea of superAdmin for a future time if we setup more administrative functionality in reearth.